docs/vpm-spec.md

```diff
@@ -0,0 +1,462 @@
+
+### 4.1. Overall Structure of VPM Package
+The VPM Package is a directory that contains the VPM Package Manifest file
+(named `package.json`) and the entries of the package.
+
+VPM Package **MUST** include file named `package.json` in the root directory of the package.
+VPM Package consists of files and directories,
+and they will be recognized as contents of the UPM Package by Unity Editor.
+
+VPM Package **SHOULD NOT** contain the file-system entries other than files and directories such as symbolic links and device files.
+
+All entries of the VPM Package **MUST** name with Unicode [[UNICODE]] characters
+and **MUST NOT** include any characters that are not allowed in the file system.
+All entries of the VPM Package **SHOULD NOT** use characters other than ASCII Alphabets, 
+Digits, Hyphen (`-`), Underscore (`_`), Space (` `), Dot (`.`), and Parentheses (`(`, `)`).
+Other characters can cause problems in some environments.
+
+> [!NOTE]
+> 
+> The rules about the file name are not provided in the [[VCC Docs]]
+>, and this is completely based on the vrc-get developers' research and opinion.
+> 
+> The official VCC uses .NET, so they cannot handle file names with non-UNICODE characters.
+> Therefore, we wrote the rule that entry name MUST name with Unicode characters.
+> 
+> For non-UTF-8 environments, Unity is unstable with non-ASCII characters.
+> Therefore, we wrote the rule that entry name SHOULD NOT use characters other than ASCII characters.
+> 
+> For some OSes like Windows, some characters are not allowed in the file name.
+> Therefore, we wrote the rule that entry name SHOULD NOT use characters other than specified characters.
+
+### 4.2. Structure of VPM Package Manifest
+VPM Package Manifest is a [[JSON]] document that describes the package information.
+The document **MUST** be named `package.json` and located in the root directory of the VPM Package.
+The document **MUST** be encoded in UTF-8 [[UNICODE]].
+VPM Package Manifest is a subset of the UPM Package Manifest.
+Some fields are required, and some fields are optional.
+Additional fields are allowed, so the VPM Client **SHOULD NOT** cause error with them.
+Some VPM Clients **MAY** use the additional fields for their purposes.
+
+#### 4.2.1. Basic Fields
+The VPM Package Manifest **MUST** include the following two fields.
+
+##### 4.2.1.1. `name` (string)
+The `name` field shows the globally unique ID of the package.
+The type of this field **MUST** be a string.
+
+The package ID is used to identify single package, so this **MUST** be unique among all packages in the VPM Repository
+and **SHOULD** be unique among all packages in the VPM ecosystem.
+VPM client will treat packages with same ID as same package.
+
+The Package ID **SHOULD** be prefixed with reverse domain name of the package author to be unique.
+
+The package ID **SHOULD** be in lowercase and **MUST NOT** contain any characters other than ASCII Alphabets, Digits, Hyphen (`-`), and Dot (`.`).
```

Comment:
Underscore('_') should be allowed.
There are Unity official package which named [`com.unity.toolchain.linux-x86_64`](https://docs.unity3d.com/Packages/com.unity.toolchain.linux-x86_64@2.0/manual/index.html).

Reply:
I found the [Unity official information about the naming convention](https://docs.unity3d.com/6000.2/Documentation/Manual/cus-naming.html).